### ISSUE TITLE
How to visualize the output of one of two merged layers ? 

### ISSUE THREAD

**Issue by Shlomke** [fchollet/keras#4007]
So for example in this code : 

`from keras.layers import Merge

left_branch = Sequential()
left_branch.add(Dense(32, input_dim=784))

right_branch = Sequential()
right_branch.add(Dense(32, input_dim=784))

merged = Merge([left_branch, right_branch], mode='concat')

final_model = Sequential()
final_model.add(merged)
final_model.add(Dense(10, activation='softmax'))`

If I want to see the output of the right branch model given a test input **(before the merge layer)** how to do that ?
Thanks

**Comment by IraZarI** [comment]
Would it be possible to use my code from #4006 ?

**Comment by Shlomke** [comment]
I do not know how exactly, 
To be more specific : what is the correct layer index  I should use ?

**Comment by IraZarI** [comment]
i dont know =) try it out. use weights=model.get_weights() to determine where you "are"

**Comment by Shlomke** [comment]
update: I used the get_weights() command. Then built a new model with the same architecture (as the branch I wanted to test) and used the correct weights.
